cli/plasmo/src/features/extension-devtools/common-path.ts

```diff
@@ -35,7 +35,7 @@ export const getCommonPath = (
 
   const distDirectoryName = `${target}-${
     process.env.NODE_ENV === "production" ? "prod" : "dev"
-  }`
+  }${tag ? '-' + tag : ''}`
```

Comment:
I think we would replace the dev/prod entirely if a tag is present. The idea is that `prod` and `dev` are technically build tag. You can use the tag flag for both dev and prod bundle, it just influence the naming of this directory.